pkpdapp/pkpdapp/templates/base/head.html

```diff
@@ -46,8 +46,8 @@
                 <a class="dropdown-item" href={% url 'dosed_pk_model-detail' m.id
                   %}>{{m.pharmacokinetic_model.name}}</a>
                 {% endfor %}
-                <a class="dropdown-item" href="{% url 'dosed_pk_model-add-to-project' current_project.id %}"> 
-                  Create model.. 
+                <a class="dropdown-item" href="{% url 'dosed_pk_model-add-to-project' current_project.id %}">
+                  Create model..
```

Comment:
Can you please add a similar "Add dataset" action link for the dataset dropdown list?

Reply:
Done